src/browser/components/TabBar.jsx

```diff
@@ -81,10 +81,24 @@ class TabBar extends React.Component {
   }
 
   renderAddTeamButton() {
+    var tabButton = {
+      border: 'none',
+      fontSize: '20px',
+      height: '35px',
+      padding: '0 0 4px 0',
+      width: '40px',
+      color: '#999',
+      fontWeight: 'bold',
+      margin: '6px 0 0',
+      borderRadius: '2px 2px 0 0',
+      outline: 'none'
+    };
+
     return (
       <Button
         id='tabBarAddNewTeam'
         onClick={this.props.onAddServer}
+        style={tabButton}
         bsStyle='tabButton'
```

Comment:
@jasonblais To create such a Tooltip, we can jsut add a title property here e.g.:
```javascript
title='Add a new server'
```